spec/runner.html

```diff
@@ -1,4 +1,4 @@
-<!DOCTYPE html>
+﻿<!DOCTYPE html>
```

Comment:
This is because the added UTF-8 header, I don't know wether this is problem or not. It was placed when Visual Studio saved the file, and I have no idea how I could avoid it.